spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe ApplicationHelper, type: :helper do
+  describe "#current_user" do
+    it "returns the username" do
```

Comment:
this test doesn't feel like it's testing much. i think it would be a more solid test if we set `session["username"]` and then called `helper.current_user`

```
describe "#current_user" do
  it "returns the username" do
    session["username"] = "Test User"
    expect(helper.current_user).to eq "Test User"
  end
end
```

Reply:
Good idea